frontend/src/components/read_more.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+    export let currentDescriptionLength;
+    export let mediaDescription;
+    export let initialDescriptionLength;
+</script>
+
+{#if currentDescriptionLength <= initialDescriptionLength && mediaDescription.length > initialDescriptionLength}
+    <p>{mediaDescription.slice(0, currentDescriptionLength)}
+        <a href="" class="cursor-pointer"
```

Comment:
Can we make the `... read more` a other color like blue?

`<a href="" class="cursor-pointer text-blue-400"`

Reply:
Implemented in the newest commit